int32 nLinear
int32 nAngular
int8 byResetOdometry
int8 byResetAngle
int8 byResetAlarm

int32 nPosition #위치 Data
